README.md

```diff
@@ -161,12 +161,54 @@ component, you can specify `supportsDataTestProperties` on the class:
 export default Ember.Component({
   tagName: '',
   supportsDataTestProperties: true
-})
+});
 ```
 
 `supportsDataTestProperties`, like `data-test-*` properties, will be stripped
 from the build.
 
+#### With splattributes
+Splattributes of angle brackets invocations, introduced in Ember.js 3.11 or
```

Comment:
they were introduced quite a bit earlier already. Ember 3.11 only made them forward element modifiers too now, while previously it was *only* attributes.

Reply:
I see. I looked back and it turns out angle brackets were added in [3.4.0](https://github.com/emberjs/ember.js/releases/tag/v3.4.0) (I had no idea!). Will make the change.

Reply:
Should be good now!